CONTRIBUTING.md

```diff
@@ -48,14 +48,14 @@ Issues can be submitted by anyone - seasoned developers or newbies.
     and
     [here to practice](http://www.markdowntutorial.com/).
 
-    *   If you are adding a new project/organisation to the README, make sure
+    *   If you are adding a new project/organization to the README, make sure
         it's listed in alphabetical order.
-    *   If you are adding a new organisation, make sure you add an organisation
-        label to the organisation name. This would help distinguish projects
-        from organisation projects.
+    *   If you are adding a new organization, make sure you add an organization
+        label to the organization name. This would help distinguish projects
+        from organisation ones.
```

Comment:
To be more clear and consistent.

```suggestion
        label to the organization name. This would help distinguish projects
        from organizations.
```

Reply:
Done.